python/semantic_kernel/connectors/ai/ollama/services/ollama_chat_completion.py

```diff
@@ -186,11 +186,6 @@ async def _inner_get_streaming_chat_message_contents(
             settings = self.get_prompt_execution_settings_from_settings(settings)
```

Comment:
Do we still need to do this?

Reply:
Yeah, if someone does call the _inner method we should doublecheck.